### DISCUSSION TITLE
Expected identifier but found "peernot" warning on vite build

### DISCUSSION THREAD

**Discussion by orelhindi-abra** [tailwindlabs/tailwindcss#14895] [Help]
**What version of Tailwind CSS are you using?**

v3.4.13

**What build tool (or framework if it abstracts the build tool) are you using?**

Vite v5.4.10

**What version of Node.js are you using?**

v20.17.0

**What browser are you using?**

Chrome,

**What operating system are you using?**

Linux

**Reproduction URL**

Its not a project bug but build issue

**Describe your issue**

When building the project with vite build for production, the following warning appears:

```
▲ [WARNING] Expected identifier but found "peernot(" [css-syntax-error]

    <stdin>:1825:1:
      1825 │ .peernot(:hover) ~ .peer-\[not\(\:hover\)\]\:hidden {
           ╵  ~~~~~~~~
```
It seems that during the build process, Tailwind CSS is generating a class .peernot(:hover), which results in this minification error.

I didnt write this class anywhere in my app
any suggestion? 

thanks!

**Comment by wongjn** [comment]
Do you have any `content` file globs that covers other code?

It seems like Tailwind has found a class `peer-[not(:hover)]:hidden` which seems to be the cause of the error. This seems like a mistaken class, as I would believe it should be `peer-[:not(:hover)]:hidden`.

**Comment by orelhindi-abra** [comment]
found the issue thanks
had a typo ` peer-[not(:hover)]` instead of `  peer-[:not(:hover)]`
